test/policy_cq_config.hcl

```diff
@@ -0,0 +1,24 @@
+// Configuration AutoGenerated by CloudQuery CLI
+cloudquery {
+
+  provider "aws" {
+    source  = "cloudquery/aws"
+    version = "latest"
+  }
+
+  // Can be configured via CLI variables
+  connection {
+    dsn = "host=localhost user=postgres password=pass database=postgres port=5432 sslmode=disable"
+  }
+}
+
+// All Provider Configurations
+provider "aws" {
+  configuration {
+    max_retries = 10
+    max_backoff = 90
+  }
+  resources = [
+    "*",
+  ]
+}
```

Comment:
add policy section below

Reply:
I don't think we actually need a policy section here, since the tests don't use it :)
https://github.com/cloudquery/.github/pull/88